app/services/beta/sort_stories.rb

```diff
@@ -0,0 +1,35 @@
+class Beta::SortStories
+  def initialize(sort_params)
+    @story = Story.find(sort_params[:id])
+    @new_position = sort_params[:new_position]
+    @new_state = sort_params[:state]
+    @position = sort_params[:position]
+  end
+
+  def call
+    @story.update(new_position: @new_position, state: @new_state, position: @position)
```

Comment:
What do you think moving this line into a method also? I believe it would show better the intentions.

```ruby
def call
  update_current_story
  update_other_stories_positions
  stories
end
```